pallets/stateful-storage/src/benchmarking.rs

```diff
@@ -32,13 +35,13 @@ benchmarks! {
 		let s in 1 .. T::MaxItemizedBlobSizeBytes::get()- 1;
 		let provider_msa_id = 1u64;
 		let delegator_msa_id = 2u64;
-		let schema_id = 1u16;
+		let schema_id = ITEMIZED_SCHEMA;
 		let caller: T::AccountId = whitelisted_caller();
 		let payload = vec![0u8; s as usize];
 
-		assert_ok!(create_schema::<T>(PayloadLocation::Itemized)); // TODO change this to itemized
+		// assert_ok!(create_schema::<T>(PayloadLocation::Itemized)); // TODO change this to itemized
```

Comment:
we can delete this and also remove the create_schema helper function looks like its not used?

Reply:
Hmmm... okay, I guess `create_schema` is necessary when actually running benchmarks (since it just failed on me locally). Updating...

Reply:
Fixed.